### DISCUSSION TITLE
Lesson 4 data.link and facucet is different from the lesson

### DISCUSSION THREAD

**Discussion by medhananda** [Cyfrin/foundry-full-course-cu#993] [Q&A]
The "data.link" site looks different from the one in the lesson, but more importantly, I see the data feed contract for sepolia and the faucet is for sepolia. 
Is this correct or is there a way to find it for Kovan? 
I am planning to use it as is on the sites, is that ok, or will it be an issue in running?
- Anu

**Comment by 0xusmanf** [comment]
> The "data.link" site looks different from the one in the lesson, but more importantly, I see the data feed contract for sepolia and the faucet is for sepolia.

That is because the tutorial is a bit old now, and the Chainlink site keeps updating frequently.

> Is this correct or is there a way to find it for Kovan?

Koven is no longer supported, so its better to use Sepolia.

> I am planning to use it as is on the sites, is that ok, or will it be an issue in running?

If you use Sepolia then there won't be any network issues in the near future.

**Reply by medhananda** [reply]
Thank you,
